### ISSUE TITLE
[Feature Request] Dm thingy

### ISSUE THREAD

**Issue by AvaneeshG** [slow/nitro-sniper#337]
### Is your feature request related to a problem?

If i don't send a dm after claiming i am either banned or the code will be revoked

### Describe the solution you'd like

I would like for me(the acc or the alt) to say "legit"/"thx" whenever it snipes a nitro code in dms this could also included my last bug report of message randomiser

### How will this be beneficial to the sniper

Message randomiser and also you don't have to worry about being catched and ppl saying "alt", "nitro sniper"...

### Please provide an example of usage of this feature

Would be good in dm gws

**Comment by Reverrb** [comment]
Could you maybe send your script? Because there is a section specifically for sending a dm to the host

**Comment by Lumi6x9** [comment]
'git' is not recognized as an internal or external command,
operable program or batch file. please tell why its happening

**Comment by mrcorpz** [comment]
@Lumi6x9 install git

**Comment by AvaneeshG** [comment]
Example of this https://media.discordapp.net/attachments/922979555386490911/931020760103923732/Screenshot_2022-01-13-08-33-18-00_fa4ea8e8aeb69dcd3ab969e8fd3d4310.jpg

**Comment by AvaneeshG** [comment]
SLOW ADD DIS RN

**Comment by Reverrb** [comment]
i meant as in the code, but you lucky as hell, all the codes have been taken

**Comment by Reverrb** [comment]
Yeah, theres nothing for this, but what you could do is make the time between messages longer which is what I did

**Comment by AvaneeshG** [comment]
@Reverrb your a dumbass what if I say "legit" even after he didn't even give me a code lmao instant ban

**Comment by Reverrb** [comment]
@AvaneeshG, hence why i said to make the time between messages longer. If you don't understand this, i can't help you

**Comment by discordtwilight** [comment]
> Example of this https://media.discordapp.net/attachments/922979555386490911/931020760103923732/Screenshot_2022-01-13-08-33-18-00_fa4ea8e8aeb69dcd3ab969e8fd3d4310.jpg

You know you dont have to nitro snipe in my server
